enum/rechnungstyp/rechnungstyp_edimapping.go

```diff
@@ -0,0 +1,55 @@
+// This maps the EDIFACT Values to the Rechnungstyp and vice versa
+// nolint:dupl // linter thinks the mapping is a duplicate to jsonenums
+package rechnungstyp
+
+import (
+	"fmt"
+)
+
+// EdiValue outputs the EDIFACT representation of the enum (AJT 4465)
+func (r Rechnungstyp) EdiValue() (string, error) {
+	s, ok := _RechnungstypValueToEdi[r]
+	if ok {
+		return s, nil
+	}
+	return "", fmt.Errorf("could not map %s", r)
+}
+
+// FromEdi maps the EDIFACT code to the enum
+func (r *Rechnungstyp) FromEdi(value string) error {
+	if v, ok := _RechnungstypEdiToValue[value]; ok {
+		*r = v
+		return nil
+	}
+	return fmt.Errorf("could not read %s", value)
+}
+
+var (
+	_RechnungstypEdiToValue = map[string]Rechnungstyp{
+		"ABR": ABSCHLUSSRECHNUNG,
+		"ABS": ABSCHLAGSRECHNUNG,
+		"JVR": TURNUSRECHNUNG,
+		"MVR": MONATSRECHNUNG,
+		"WIM": WIMRECHNUNG,
+		"ZVR": ZWISCHENRECHNUNG,
+		"13I": INTEGRIERTE_13TE_RECHNUNG,
+		"13R": ZUSAETZLICHE_13TE_RECHNUNG,
+		"MMM": MEHRMINDERMENGENRECHNUNG,
+		"MSB": MSBRECHNUNG,
+		"NAP": KAPAZITAETSRECHNUNG,
+	}
+
+	_RechnungstypValueToEdi = map[Rechnungstyp]string{
+		ABSCHLUSSRECHNUNG:          "ABR",
+		ABSCHLAGSRECHNUNG:          "ABS",
+		TURNUSRECHNUNG:             "JVR",
+		MONATSRECHNUNG:             "MVR",
+		WIMRECHNUNG:                "WIM",
+		ZWISCHENRECHNUNG:           "ZVR",
+		INTEGRIERTE_13TE_RECHNUNG:  "13I",
+		ZUSAETZLICHE_13TE_RECHNUNG: "13R",
+		MEHRMINDERMENGENRECHNUNG:   "MMM",
+		MSBRECHNUNG:                "MSB",
+		KAPAZITAETSRECHNUNG:        "NAP",
+	}
```

Comment:
Das sehe ich analog zu meinem Kommentar in https://github.com/Hochfrequenz/go-bo4e/pull/108#pullrequestreview-954848863 nicht als Teil der BO4E-Repo.

Reply:
Das hatte ich leider verpasst. An vielen Stellen bildet ja aber BO4E genau den Wertevorrat und das Modell aus den Formaten ab. Insofern sehe ich das schon als sehr stark verbunden an.

Reply:
OK. Dann machen wir das vielleicht wirklich separat

Reply:
getrennte PRs für getrennte Features sind eh was feines ;)